src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={cn({ 'is-active': tab.id === selectedTab.id })}
+            data-cy="Tab"
+          >
+            <a
+              onClick={
+                (event) => {
+                  if (tab !== selectedTab) {
+                    onTabSelected(tab);
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here